hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -50,6 +50,23 @@
 
 CORS_ALLOW_CREDENTIALS = True
 
+RECAPTCHA_DOMAIN = "www.recaptcha.net"
+
+RECAPTCHA_TEST_PUBLIC_KEY = "6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI"
+RECAPTCHA_TEST_PRIVATE_KEY = "6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe"
+
+if DEBUG or IN_TESTING:
```

Comment:
Ok my bad, I forgot that `IN_TESTING` doesn't get changed to true until after this file finishes running, should probably have made that an environment variable originally.

You can change the keys below back to `os.environ.get("RECAPTCHA_PUBLIC_KEY", RECAPTCHA_TEST_PUBLIC_KEY)` etc. I tested, and the systems check framework will actually prevent the site from running, so nobody is at risk of accidentally using the test keys in production.